client/signup/utils.js

```diff
@@ -146,19 +146,19 @@ export function getThemeForDesignType( designType ) {
 }
 
 export function getThemeForSiteType( siteType ) {
-	if ( 'business' === dasherize( siteType ) ) {
+	if ( siteType && siteType === get( allSiteTypes, 'business.value', '' ) ) {
```

Comment:
We can probably leave these hardcoded to their siteType IDs for the time being (i.e. `'business' === siteType`).